packages/libs/registry-sdk/docs/api.html

```diff
@@ -3,9 +3,310 @@
   <head>
     <meta charset="utf-8" />
     <script type="module" src="https://unpkg.com/rapidoc/dist/rapidoc-min.js"></script>
+    <script type="text/javascript" src="https://unpkg.com/ethers@5.8.0/dist/ethers.umd.js">
+    </script>
+    <style>
+      .metamask-button {
+        background-color: #f6851b;
+        color: white;
+        border: none;
+        padding: 8px 16px;
+        border-radius: 4px;
+        cursor: pointer;
+        display: flex;
+        align-items: center;
+        font-weight: bold;
+      }
+      .metamask-button img {
+        margin-right: 8px;
+        height: 20px;
+      }
+      .metamask-container {
+        padding: 16px;
+        border: 1px solid #ddd;
+        border-radius: 4px;
+        margin-bottom: 16px;
+      }
+      .metamask-status {
+        margin-top: 8px;
+        font-size: 14px;
+      }
+      .connected {
+        color: green;
+      }
+      .error {
+        color: red;
+      }
+    </style>
   </head>
 
   <body>
-  <rapi-doc spec-url="../src/generated/openapi.json" theme="light"> </rapi-doc>
+    <rapi-doc 
+      spec-url="../src/generated/openapi.json" 
+      theme="light"
+      render-style="read"
+      show-header="false"
+      allow-authentication="true"
+      allow-server-selection="true"
+    >
+      <div slot="auth" class="metamask-container">
+        <h3>SIWE Authentication with Metamask</h3>
+        <div>
+          <button id="connectMetamask" class="metamask-button">
+            <img src="https://images.ctfassets.net/clixtyxoaeas/4rnpEzy1ATWRKVBOLxZ1Fm/a74dc1eed36d23d7ea6030383a4d5163/MetaMask-icon-fox.svg" alt="Metamask logo" />
+            Connect with Metamask
+          </button>
+          <div id="metamaskStatus" class="metamask-status"></div>
+        </div>
+        <div id="authDetails" style="margin-top: 16px; display: none;">
+          <p>Connected Address: <span id="connectedAddress"></span></p>
+          <div style="margin-bottom: 16px;">
+            <label for="domainInput" style="display: block; margin-bottom: 8px; font-weight: bold;">Domain:</label>
+            <input 
+              id="domainInput" 
+              type="text" 
+              placeholder="Enter domain (e.g., app.example.com)" 
+              style="width: 100%; padding: 8px; border: 1px solid #ddd; border-radius: 4px;"
+            >
+            <small style="display: block; margin-top: 4px; color: #666;">Domain used in the SIWE message. Defaults to current host if empty.</small>
+          </div>
+          <button id="generateSiwe" class="metamask-button" style="background-color: #0366d6;">
+            Generate SIWE Authentication
+          </button>
+          <div id="siweStatus" class="metamask-status"></div>
+        </div>
+      </div>
+
+      <script>
+        // Implementation of SIWE functionality
```

Comment:
In-lined this code from the `siwe` package here, because there is no UMD package for `siwe`, and setting up a frontend packaging pipeline in this package just for this file to be able to compose a SIWE seemed like an excessive effort.